docs/getting-started-guides/minikube.md

```diff
@@ -131,7 +131,7 @@ kubectl get nodes
 
 Minikube contains a built-in Docker daemon for running containers.
```

Comment:
Reverse the order of the sentences. I recommend restructuring the paragraph as follows:

"Minikube contains a built-in Docker daemon for running containers, which lets you use the containers directly instead of publishing them to a registry. If you use a separate Docker daemon to build your containers, you'll have to publish your containers to a registry before Minikube can pull them.

You can use the built-in Docker daemon by running the following command:"